vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -84,31 +88,50 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
             @RequestParam("spaceParam") String spaceID, @RequestParam("title") String title,
             @RequestParam("exhibitMode") ExhibitionModes exhibitMode,
             @RequestParam(value = "customMessage", required = false, defaultValue = "") String customMessage,
+
+            @RequestParam("externalLinkImage") MultipartFile externalLinkImage,
+            @RequestParam("spacelinkImage") MultipartFile spacelinkImage,
+            @RequestParam("moduleLinkImage") MultipartFile moduleLinkImage,
             @RequestParam("exhibitLanguage") List<String> languages,
             @RequestParam("defaultExhibitLanguage") String defaultLanguage,
+
             RedirectAttributes attributes) throws IOException {
 
         ISpace startSpace = spaceManager.getSpace(spaceID);
-
+        
         Exhibition exhibition;
         if (exhibitID == null || exhibitID.isEmpty()) {
             exhibition = (Exhibition) exhibitFactory.createExhibition();
         } else {
             exhibition = (Exhibition) exhibitManager.getExhibitionById(exhibitID);
         }
+        
+        IVSImage spaceDefaultImage = spacelinkImage != null ? 
+                imageService.storeImage(spacelinkImage.getBytes(), spacelinkImage.getOriginalFilename()) : null; 
```

Comment:
If no spacelinkImage is passed in, then `spaceDefaultImage` will be set to null here and the default image will be deleted. But if the user does not select a new image on the configuration page, then the existing one should be retained, but I think the image wouldn't be passed to the controller again, would it? I think there are two operations needed, one to change the image (or keep the same) and one to delete the image (which probably needs to be an extra button/controller call?).

Reply:
In fact, I think I would not include the default images with the general configuration form but have them separate actions, so that you don't have to submit the whole exhibition configuration form to change or delete the default images, those could be ajax calls.